README.md

```diff
@@ -1,2 +1,174 @@
-# Capstone-Grp_Kanban
-In this activity, we have prepared as a team a Kanban board with a GitHub project. 
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>ES6 Loved-Movies Website</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 ES6 Loved-Movies Website (JavaScript Capstone Project) <a name="about-project"></a>
+
+**ES6 Loved-Movies Website** is an activity that we prepared as a team using a Kanban board with a GitHub project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.testim.io/blog/what-is-a-linter-heres-a-definition-and-quick-start-guide/">Linters</a></li>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add Task**
+- **Remove Task**
+- **Remove Completed Tasks**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live demo here](https://Kanyorok.github.io/Capstone-Grp_Kanban/dist)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- A code editor of your choice (like vs code or atom and so on)
+-version control system (git is preferred)
+
+### Setup
+
+- Create a new folder to store the files in.
+
+### Install
+
+Install this project with:
+
+- Open your command prompt or terminal and run.
+- git clone https://github.com/Kanyorok/Capstone-Grp_Kanban.git
+- cd Capstone-Grp_Kanban
+- npm i
+- npm start (To Launch App).
+
+
+### Usage
+
+- this is a Website Application app and I will use it to insert my upcoming projects.
+
+### Run tests
+
+To run tests, run the following command:
+
+- npm test
+- npm lint
+- npm start
+
+### Deployment
+
+You can deploy this project using:
+
+```
+  github pages
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1: Robert Kanyoro**
+
+- GitHub: [@RKanyoro](https://github.com/RKanyoro)
```

Comment:
- [ ] Kindly fix this broken link, I got 404 error. Fixing this will make your repo to be more engaging.